src/data/items.h

```diff
@@ -1852,8 +1852,13 @@ const struct Item gItems[] =
         .name = _("Sitrus Berry"),
         .itemId = ITEM_SITRUS_BERRY,
         .price = 20,
-        .holdEffect = HOLD_EFFECT_RESTORE_HP,
-        .holdEffectParam = 30,
+        #if I_SITRUS_BERRY_HEAL >= GEN_4
```

Comment:
```suggestion
        #if defined(BATTLE_ENGINE) && I_SITRUS_BERRY_HEAL >= GEN_4
```
Allows it to fall back to the vanilla behaviour in battles if you don't have Battle Engine merged in since `HOLD_EFFECT_RESTORE_PCT_HP` isn't implemented in Item Expansion.

Do double-check if this works before accepting the suggestion, just in case.